message_de_bienvenue.py

```diff
@@ -8,4 +8,24 @@ def hello_nom(nom):
             return "error"
 
     else:
-        return "error"
\ No newline at end of file
+        return "error"
+
+
+def hello_blank(string):
+    if is_blank(string):
+        return "Hello, my friend"
+    else:
+        return "error"
+
+
+def is_blank(string):
+    boolean_is_blank = True
+    if len(string) != 0:
+        i = 0
+        while i < len(string) and boolean_is_blank == True:
```

Comment:
**Compréhension :**  Moins long et plus explicite avec un for peut-être mais sinon ça marche pareil ^^